.gitignore

```
@@ -2,4 +2,5 @@ target/
 dbt_modules/
 logs/
 .DS_Store
-dbt_packages/
\ No newline at end of file
+dbt_packages/
+package-lock.yml
```

Request to also add the virtual environment to the gitignore.
```suggestion
package-lock.yml
env/
```